meta-mentor-common/classes/kernel-config-flatpak.bbclass

```diff
@@ -0,0 +1,9 @@
+FILESEXTRAPATHS_append = ":${@os.path.dirname(bb.utils.which(BBPATH, 'files/namespaces.cfg') or '')}"
+
+NAMESPACES_CFG = "${@bb.utils.contains('BBLAYERS', 'flatpak-layer', 'file://namespaces.cfg', '', d)}"
```

Comment:
Why in one class we're doing this and in the other one we're adding cfg files directly to the SRC_URI?

Reply:
@abelal, no specific reason for it. To conserve uniformity, I'll update the other bbclass like this one.

Reply:
+1